### PR TITLE
add docs to include option at ActiveModel::Serialization [ci skip]

### PR DESCRIPTION
**Pull request by rusikf**
Include option is not documented well, so i think it will be good to add docs about it.
How it works you can see in [test, which is already exists](https://github.com/rails/rails/blob/master/activemodel/test/cases/serialization_test.rb).

### CODE REVIEW

File: activemodel/lib/active_model/serialization.rb [hunk 1]
```diff
@@ -69,6 +69,37 @@ module ActiveModel
...
+  #   note.title = 'Battle of Austerlitz'
+  #   note.text = 'Some text here'
+  #
+  #   user = User.new
+  #   user.name = 'Napoleon'
+  #   user.notes = [note]
+  #
+  #   user.serializable_hash
+  #   #=> {"name" => "Napoleon"} 
+  #   user.serializable_hash(include: {notes: {only: 'title'}})
```
Comment (rafaelfranca) [inline activemodel/lib/active_model/serialization.rb]: space after the `{` and before the `}`

Comment (rusikf) [inline activemodel/lib/active_model/serialization.rb]: Fixed!  [In the same manner like here](https://github.com/rusikf/rails/blob/patch-1/activemodel/lib/active_model/serialization.rb#L71)

**Comment by robin850** [comment]
Hi @rusikf!

Thanks for your patch! What do you think about moving this inside the documentation of the `serializable_hash` method ? This option is not specific to the API of the module but to this method and it looks like it's not even mentioned in its documentation.

Also could you please squash your commits together with `git rebase -i HEAD~2` ? Just replace the last line's `pick` with `squash` and edit the commit message appropriately. Then you can force push to this branch.

**Comment by rusikf** [comment]
@robin850 , thank's for review, I fixed changes again.

**Comment by robin850** [comment]
Thanks @rusikf and sorry for the late reply! :-)

**Pull request merged**